### PR TITLE
fix: add dropdown styles for the header navigation bar

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
## Header more dropdown
image
### Hover
image
### Dropdown
image

## version switcher dropdown
image
image

### CODE REVIEW

File: src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/pydata-sphinx-theme-custom.css [hunk 1]
```diff
@@ -643,3 +610,78 @@ aside.topic,
...
+}
+
+button.btn.version-switcher__button:hover {
+  border-color: var(--ast-navbar-active-border-color);
+  color: var(--ast-navbar-hover-color);
+}
+
+.version-switcher__menu a.list-group-item:hover {
+  background-color: var(--ast-color-hover-card-background);
+  border: none;
```
Comment (jorgepiloto) [inline src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/pydata-sphinx-theme-custom.css:670]: I feel like both dropdowns (version and "more") should be have a border. Right now, the "more" dropdown has one but the version one does not.

What is your feedback on this, @mia-guo-ux ?

Comment (Revathyvenugopal162) [reply src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/pydata-sphinx-theme-custom.css:670]: I removed it according to the suggestion in [this GitHub issue](https://github.com/ansys/ansys-sphinx-theme/issues/403#issuecomment-2179523375), but I'm happy to revert it if needed.

Comment (jorgepiloto) [reply src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/pydata-sphinx-theme-custom.css:670]: Thanks for the explanation, @Revathyvenugopal162. Just wanted to double check with Mia about this.

Comment (mia-guo-ux) [reply src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/pydata-sphinx-theme-custom.css:671]: Thanks for bringing it up :) In our design system, we do not have a border for dropdown menus. Instead, we apply drop shadows to indicate the hierarchy and to distinguish it from the background. Please let me know if you have any questions.

Comment (jorgepiloto) [reply src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/pydata-sphinx-theme-custom.css:671]: Thanks for the clarification, Mia.

**Review by jorgepiloto — APPROVED** [review]
Feel free to merge, @Revathyvenugopal162

**Pull request merged**